source/beamline/SXRMB/SXRMBBrukerDetector.cpp

```diff
@@ -28,6 +28,7 @@ SXRMBBrukerDetector::SXRMBBrukerDetector(const QString &name, const QString &des
 
 	// Stuff specialized by SXRMBBrukerDetector
 	deadTimeControl_ = new AMReadOnlyPVControl("Dead Time", "mca1606-B10-03:mca1.DTIM", this);
+	allControls_->addControl(deadTimeControl_);
 	connect(deadTimeControl_, SIGNAL(connected(bool)), this, SLOT(onDeadTimeControlConnected(bool)));
```

Comment:
It is kind of important. We didn't add the deadTimeControl_ to the set. And if deadTimeControl_ is not connected, we didn't set the connectivity of the bruker detector properly.